fdmi/plugins/fdmi_sample_plugin.h

```diff
@@ -42,21 +42,14 @@
 #include <fcntl.h>
 #include <stdio.h>
 
-/* sched client conf params */
-struct sched_conf {
-	char          *local_addr;
-	char          *ha_addr;
-	char          *profile_fid;
-	char          *process_fid;
-	char          *fdmi_plugin_fid_s;
-	struct m0_fid  fdmi_plugin_fid;
-} c_params;
+/* Plugin sample conf params */
+struct m0_fsp_params {
+	char          *spp_local_addr;
```

Comment:
what does `spp` mean? Can you specify in comments please?

Reply:
ok